packages/extension/src/components/pages/SendPayment/ConfirmPayment/ConfirmPayment.tsx

```diff
@@ -132,95 +165,20 @@ export const ConfirmPayment: FC<ConfirmPaymentProps> = ({
     );
   }
 
-  const decodedMemos = fromHexMemos(memos);
-  const [currency, issuer] = token.split('-');
-
   return (
-    <PageWithReturn
-      title="Confirm Payment"
-      onBackClick={onClickBack}
-      style={{
-        height: '100%',
-        display: 'flex',
-        flexDirection: 'column',
-        justifyContent: 'space-around'
-      }}
+    <TransactionPage
+      title="Send Payment"
```

Comment:
```suggestion
      title="Submiy Payment"
```
Would this make more sense as the button is named: `Submit`?

Reply:
I don't know!